lib/hammer_cli/defaults.rb

```diff
@@ -0,0 +1,79 @@
+require 'hammer_cli/defaults_commands'
+module HammerCLI
+  class Defaults
+    DEFAULT_FILE = "#{Dir.home}/.hammer/cli.modules.d/defaults.yml"
```

Comment:
`cli.modules.d` is for hammer module configuration. This config should be in "#{Dir.home}/.hammer/defaults.yml". What dou you think? @tstrachota?